pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -7,6 +7,46 @@
             "#import": [ "base" ]
         },
 
+        "android": {
+            "#import": [ "any" ]
+        },
+        "android-arm64": {
+	           "#import": [ "any" ]
+        },
+        "android-mips": {
+            "#import": [ "any" ]
+        },
+
+        "android.21": {
+	           "#import": [ "android" ]
+	       },
+        "android.21-arm64": {
+	           "#import": [ "android.21", "android-arm64" ]
+        },
+        "android.21-mips": {
+	           "#import": [ "android.21", "android-mips" ]
+        },
+
+        "android.22": {
```

Comment:
@qmfrederik is right, the APIs are backwards compatible but there are some breaking changes when moving forward through the API levels.  For example position-independent executables is enforced for dynamically-linked exes in API-21.

I vaguely recall a post in which MIPS was mentioned as a valid Android processor; if no one is actively working on it it can be removed.

To sum up:

- Add API level 25, remove 21-23
- remove MIPS